### ISSUE TITLE
BUG: Cleanup menu items that are not applicable

### ISSUE THREAD

**Issue by mnaoumov** [mnaoumov/obsidian-root-folder-context-menu#3]
@mnaoumov 

its probably a good idea to remove these too, just to tidy it up a little!

![image](https://github.com/user-attachments/assets/5ad80081-0c05-42bb-82d2-ec65fc5ef2a6)

_Originally posted by @Lyqed in https://github.com/mnaoumov/obsidian-root-folder-context-menu/issues/1#issuecomment-2566633637_

**Comment by Lyqed** [comment]
it might also be a good idea to hide the "search in folder" as that's what the search icon is for 😺 

![image](https://github.com/user-attachments/assets/ac0f1a8c-3d4f-4169-b7ad-34ae8ea5c6b2)

**Comment by mnaoumov** [comment]
No, search will probably stay, as it doesn't break anything, it's just redundant

**Comment by mnaoumov** [comment]
@Lyqed

Fixed in https://github.com/mnaoumov/obsidian-root-folder-context-menu/releases/tag/1.4.0
